requests/cookies.py

```diff
@@ -144,11 +144,10 @@ def remove_cookie_by_name(cookiejar, name, domain=None, path=None):
     """
     clearables = []
     for cookie in cookiejar:
-        if cookie.name != name:
-            continue
-        if domain is not None and domain != cookie.domain:
-            continue
-        if path is not None and path != cookie.path:
+        bool_cookie = (cookie.name != name)
```

Comment:
@Lukasa Thanks for clearing things up. I agree though it's not `nested` if's but wouldn't it be good if multiple `if's` can be reduced to a single `if` statement. Sandi _mentioned_ to have kind of `linear` structure when overviewed which is again accomplished by this change. I agree with the issue in variables naming and apologize for the wrong naming. I will update the variable names and add the comment as mentioned by you. 
What is your feedback on the changes I mentioned in the `list_domains` function?